src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/SyncRequest.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.SYSTEM_USERNAME
+import uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.jpa.NonAssociation
+
+@Schema(description = "Sync Request")
+data class SyncRequest(
+  @Schema(description = "Prisoner number to not associate", required = true, example = "A1234BC")
+  val firstPrisonerNumber: String,
+  @Schema(
+    description = "Reason why this prisoner should be kept apart from the other",
+    required = true,
+    example = "VICTIM",
+  )
+  val firstPrisonerReason: NonAssociationReason,
+  @Schema(description = "Prisoner number to not associate", required = true, example = "D5678EF")
+  val secondPrisonerNumber: String,
+  @Schema(
+    description = "Reason why this prisoner should be kept apart from the other",
+    required = true,
+    example = "PERPETRATOR",
+  )
+  val secondPrisonerReason: NonAssociationReason,
+
+  @Schema(description = "Type of restriction, e.g. don't locate in the same cell", required = true, example = "CELL")
+  val restrictionType: NonAssociationRestrictionType,
+
+  @Schema(
+    description = "Type of restriction, e.g. don't locate in the same cell",
+    required = false,
+    example = "John and Luke always end up fighting",
+  )
+  val comment: String? = null,
+
+  @Schema(description = "Who authorised the non-association", required = false, example = "John Smith")
+  val authorisedBy: String? = null,
+) {
+  fun toNewEntity(): NonAssociation {
+    return NonAssociation(
+      id = null,
+      firstPrisonerNumber = firstPrisonerNumber,
+      firstPrisonerReason = firstPrisonerReason,
+      secondPrisonerNumber = secondPrisonerNumber,
+      secondPrisonerReason = secondPrisonerReason,
+      restrictionType = restrictionType,
+      comment = comment ?: "NO COMMENT",
+      authorisedBy = authorisedBy ?: SYSTEM_USERNAME,
+    )
+  }
```

Comment:
[Same comment](https://github.com/ministryofjustice/hmpps-non-associations-api/pull/38/files#r1270666533) to the corresponding migration method: Watch out because this would create an open non-association which may not be what we want if the non-association synced from NOMIS is actually closed.

This is _possibly_ fine for syncing _new_ non-associations, but I would be more explicit nevertheless, something to bear in mind anyway.